src/OSM_dataTool.js

```diff
@@ -45,13 +44,20 @@ module.exports = function ({ routes, ways, assumeFirstWayIsStart, mapProperties,
                 }
             })
         } catch (error) {
-            debug(`Error: ${error.message}`)
+            debug(`Error: ${error.extractor_error || error.message}`)
+            log_file.push(Object.assign({
+                id: current_route.id,
+                error_log: error.extractor_error ? error : "not controlled"
```

Comment:
Isn't `error` a more concise name?

```suggestion
                error: error.extractor_error ? error : "not controlled"
```